api/test/integration/test_logtest_endpoints.tavern.yaml

```diff
@@ -0,0 +1,151 @@
+---
+test_name: PUT /logtest
+
+marks:
+  - base_tests
+
+stages:
+
+  # Run logtest with an invalid json body (with an invalid key)
+  - name: Run logtest with an invalid body
+    request:
+      verify: False
+      url: "{protocol:s}://{host:s}:{port:d}/logtest"
+      method: PUT
+      headers:
+        Authorization: "Bearer {test_login_token}"
+        content-type: application/json
+      json:
+        invalid_key: "token"
+    response:
+      status_code: 400
+
+  - name: Run logtest with an invalid token
```

Comment:
Why this returns a 200 response with all ok if it is using an invalid token?

Reply:
It is using a token with no logtest session. In the response, the new token for the new session is shown.
I have changed the test name in order to make it easier to understand.